### ISSUE TITLE
Umbilical cord prolapse

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7462]
**Preferred term label:**
Umbilical cord prolapse

**Synonyms**
UCP; cord prolapse; 

**Definition (free text, please give PubMed ID)**
An abnormal condition that occurs when the umbilical cord exits the cervical opening before the fetal presenting part (PMID: 31194398)

**Comment**
(It is diagnosed by seeing/palpating the prolapsed cord outside or within the vagina in addition to abnormal fetal heart rate patterns. PMID: 30174462)

**Parent term (use hpo.jax.org/app)**
Abnormality of the umbilical cord HP:0010881

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by umberagarwal** [comment]
I feel that this is a clinical obstetric diagnosis and don't see its relevance

In terms of the parent term - abnormality of UC, the cord prolapse per se is not an abnormality. 

https://orcid.org/my-orcid?orcid=0000-0001-7600-5359

**Comment by pnrobinson** [comment]
I have added this term as a child of Abnormality of the abdominal wall HP:0004298

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000329
- label: Umbilical cord prolapse
- parents: [Abnormality of the umbilical cord (HP:0010881)](https://hpo.jax.org/app/browse/term/HP:0010881)
- pmid: PMID:31194398
- def: The presence of the umbilical cord (UC) between the fetal presenting part and the cervix, regardless of the membrane status (intact or ruptured). Descent of the UC through the cervix is essential for diagnosing cord prolapse. It can be either overt (past the presenting part) or occult (alongside the presenting part).
- comment: 
- orcid: orcid:0000-0002-6670-9157
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7462
- synonyms:
